lib/puppet/type/foreman_smartproxy.rb

```diff
@@ -52,6 +52,10 @@ def is_to_s(value)
     newvalues(URI.regexp)
   end
 
+  newparam(:instance_id) do
+    desc 'Instance id of the smartproxy'
+  end
```

Comment:
I don't see a provider part to this. Where does this actually have an effect?

Reply:
It is used here https://github.com/theforeman/puppet-foreman_proxy/pull/641/files#diff-f9877574c121f639214a4b7e1b0d6ecf4f988fa27f57ba33cf75717c561f54e7R13

Reply:
Right, changed.